src/modelmanager.cpp

```diff
@@ -720,6 +739,18 @@ Status ModelManager::loadConfig(const std::string& jsonFilename) {
     if (!status.ok()) {
         IF_ERROR_NOT_OCCURRED_EARLIER_THEN_SET_FIRST_ERROR(status);
     }
+
+    // Reading metric config only once per server start
+    if (!this->metricConfigLoadedOnce) {
+        status = loadMetricsConfig(configJson);
+        SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Reading metric config only once per server start");
+        this->metricConfigLoadedOnce = true;
+
+        if (!status.ok()) {
+            IF_ERROR_NOT_OCCURRED_EARLIER_THEN_SET_FIRST_ERROR(status);
+        }
+    }
```

Comment:
Maybe add log here (debug level) indicating that reading monitoring config was skipped? @rasapala